UnitsNet/Scripts/Include-GenerateQuantityTypeSourceCode.ps1

```diff
@@ -43,12 +43,12 @@ function GenerateQuantityTypeSourceCode($quantityNames)
 
 namespace UnitsNet
 {
-	/// <summary>
-	///     Lists all generated quantities with the same name as the quantity struct type,
-	///     such as Length, Mass, Force etc.
-	///     This is useful for populating options in the UI, such as creating a generic conversion
-	///     tool with inputValue, quantityName, fromUnit and toUnit selectors.
-	/// </summary>
+    /// <summary>
+    ///     Lists all generated quantities with the same name as the quantity struct type,
+    ///     such as Length, Mass, Force etc.
+    ///     This is useful for populating options in the UI, such as creating a generic conversion
+    ///     tool with inputValue, quantityName, fromUnit and toUnit selectors.
+    /// </summary>
```

Comment:
Good thing to catch whitespace issues like this

Reply:
I personally prefer tabs, but consistency rules the day :)